### PR TITLE
Add Pending Approvals Special Page

### PR DESCRIPTION
**Pull request by malberts**
For #23 

Current state:
Screenshot_20240701_172406

Page titles are links.

### CODE REVIEW

**Comment by malberts** [comment]
Currently there seems to be a problem with the categories retrieved via `DatabasePendingApprovalRetriever`. All 3 of the pages in the screenshot have multiple categories.

File: src/EntryPoints/Specials/SpecialPendingApprovals.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+	/**
+	 * @param string[] $cells
+	 */
+	private function createTableRow( array $cells, string $cellType = 'td' ): string {
+		$rowContent = '';
+		foreach ( $cells as $cell ) {
+			$rowContent .= Html::rawElement( $cellType, [], $cell );
+		}
+		return Html::rawElement( 'tr', [], $rowContent );
```
Comment (JeroenDeDauw) [inline src/EntryPoints/Specials/SpecialPendingApprovals.php:63]: Often you can make these more elegant with implode + higher-order functions

Comment (JeroenDeDauw) [reply src/EntryPoints/Specials/SpecialPendingApprovals.php:63]: This is totally fine tho!

**Comment by JeroenDeDauw** [comment]
I've started wondering is showing the last edit time and user makes sense. Perhaps it makes more sense to show when the page was unapproved and by who (or automatic). Or maybe both sets of info. Let's just start with what we already have implemented.

**Pull request merged**